app.rb

```diff
@@ -0,0 +1,79 @@
+require 'sinatra'
+require 'data_mapper'
+require 'rack-flash'
+
+env = ENV['RACK_ENV'] || 'development'
+
+DataMapper.setup(:default, "postgres://localhost/chitter_#{env}")
+
+require './lib/cheet'
+require './lib/user'
+
+DataMapper.finalize
+DataMapper.auto_upgrade!
+
```

Comment:
I really like the modular way to create a sinatra application over the classical way. Look up the differences and use the one which is more widely accepted.